client/src/templates/Challenges/classic/show.tsx

```diff
@@ -238,12 +238,11 @@ function ShowClassic({
   // made during the build (at least twice!). It should be either a prop or
   // computed from challengeType
   const showPreview =
-    (challengeType === challengeTypes.html ||
-      challengeType === challengeTypes.modern ||
-      challengeType === challengeTypes.multifileCertProject ||
-      challengeType === challengeTypes.multifilePythonCertProject ||
-      challengeType === challengeTypes.python) &&
-    block !== 'learn-introductory-javascript-by-building-a-pyramid-generator';
+    challengeType === challengeTypes.html ||
```

Comment:
Can we do something like this?
```js
const isValid = [
    challengeTypes.html,
    challengeTypes.modern,
    challengeTypes.multifileCertProject,
    challengeTypes.multifilePythonCertProject,
    challengeTypes.python
].some(type => type === challengeType);
```

Reply:
Sure. `[].includes()` is a little more direct, though. I'll push a commit.